contracts/staking/stakeManager/StakeManagerStorageExtension.sol

```diff
@@ -14,4 +17,7 @@ contract StakeManagerStorageExtension {
     uint256 public maxRewardedCheckpoints;
     // increase / decrease value for faster or slower checkpoints, 0 - 100%
     uint256 public checkpointRewardDelta;
+
+    IERC20 public tokenLegacy;
+    IPolygonMigration public migration;
```

Comment:
why do we need these again here?

Reply:
StakeManagerStorageExtension represent the "end" of all storage slots (other than stakemanager itself).
If we don't add here we get slot conflicts.

Reply:
ah, i hallucinated this being added on stakeManager as well haha, verified it appends correctly to storage here 👌